docs-src/cloud/limits.md

```diff
@@ -132,6 +132,12 @@ When creating custom Search Attributes in Temporal Cloud, the attribute names mu
 
 For more information on custom Search Attributes see [Custom Search Attributes limits](/visibility#custom-search-attributes).
 
+### Visibility API Rate Limit
+
+**What is the rate limit for requests to the Visibility APIs?**
+
+The default rate limit for requests to the Visibility APIs is 10 requests per second.
```

Comment:
This is a per-namespace limit; does that need to be explicit here?

Reply:
I think that's an important clarification to include.